homeassistant/components/sensor/google_travel_time.py

```diff
@@ -67,7 +67,7 @@
         }))
 })
 
-TRACKABLE_DOMAINS = ['device_tracker', 'sensor', 'zone']
+TRACKABLE_DOMAINS = ['device_tracker', 'sensor', 'zone', 'person']
```

Comment:
Why is it necessary to specify which things can be tracked? Shouldn't anything with a lat/long work?

Reply:
The overall design is a little weird. I was hoping to resolve this in the TravelTime component after the waze refactor pr was merged.